### PR TITLE
Adding Swift Package Manager manifest file.

### PR DESCRIPTION
**Pull request by McProb**

### CODE REVIEW

**Comment by hunnefeldtyler** [comment]
@indragiek talked to you on Twitter, my coworker Rob ended up adding the SPM compatibility rather than myself. Let us know if anything looks off!

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "InAppViewDebugger",
+            targets: ["InAppViewDebugger"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+    ],
```
Comment (indragiek) [inline Package.swift]: nit: drop this boilerplate

File: Package.swift [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import PackageDescription
+
+let package = Package(
+    name: "InAppViewDebugger",
+    platforms: [
+        .iOS(.v13),
+        .watchOS(.v6)
+    ],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
```
Comment (indragiek) [inline Package.swift]: nit: remove comment

File: Package.swift [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+            name: "InAppViewDebugger",
+            targets: ["InAppViewDebugger"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages this package depends on.
```
Comment (indragiek) [inline Package.swift]: nit: remove comment

**Review by indragiek — CHANGES_REQUESTED** [review]
Just a couple nits to make this more concise, thanks to both of you for contributing this!

**Comment by McProb** [comment]
Happy to make the file more concise.  I think it will be important to note that using package manager with this framework requires Xcode 12 or higher in order to include the xcassets.  I think this is good to go.  Thanks again!

**Review by indragiek — APPROVED** [review]

**Comment by indragiek** [comment]
Thank you @McProb!

**Pull request merged**